### ISSUE TITLE
extend website simple search to cover extensions once extension pages are present

### ISSUE THREAD

**Issue by ValWood** [pombase/website#1652]
From
https://github.com/pombase/website/issues/473

**Comment by ValWood** [comment]
@kimrutherford  says later in the ticket that this will  probably happen automatically

**Comment by kimrutherford** [comment]
> @kimrutherford says later in the ticket that this will probably happen automatically

It needs some work though because the results might not be what the user expects when using the current implementation.

**Comment by ValWood** [comment]
maybe another classifier/facet "matching extensions"

**Comment by ValWood** [comment]
similar to the query builder ticket
https://github.com/pombase/website/issues/1653
I can't think of a useful use case for GO, except for "phases" and "responses".
These would also be applicable to annotation type "Modification"
e.g.


<img width="604" alt="Screenshot 2021-12-30 at 18 16 25" src="https://user-images.githubusercontent.com/7359272/147777970-9678268e-b139-46a9-ad5b-495c0b7933ce.png">

and gene expression:
<img width="604" alt="Screenshot 2021-12-30 at 18 16 25" src="https://user-images.githubusercontent.com/7359272/147778037-01c45fb7-c876-4581-a101-48427a178895.png">

So we could begin with these (phase and response)?

There are term pages already, but they do not  (usually) have any direct annotations. e.g.

https://www.pombase.org/term/GO:0000087
https://www.pombase.org/term/GO:0072713

but occasionally, they do like:
https://www.pombase.org/term/GO:0009408

and sometimes processes are used in extensions (cytokinesis, conjugation), although not so many are allowed.


Maybe the first step  should be to display the use in extensions in a separate section on the term pages?)

related ticket:
https://github.com/pombase/website/issues/1815
https://github.com/pombase/website/issues/473 (in fact this seems more of a complete ticket)

**Comment by kimrutherford** [comment]
> Maybe the first step should be to display the use in extensions in a separate section on the term pages?
>
>  #473 (in fact this seems more of a complete ticket)

Yep, I started work on this a while ago and it opened a can of worms.  Several parts of the site broke and we weren't sure that extension sections would make sense to users.  It needs a bit of thought and discussion.

**Comment by ValWood** [comment]
Removed 'next' added 'discuss'. We can chat about this one once the decks are cleared a bit.
